lib/pages/searchScreen.dart

```diff
@@ -0,0 +1,320 @@
+            });
+            getLectureList();
+          },
+          items: departmentList.map<DropdownMenuItem<String>>((String value) {
+            return DropdownMenuItem<String>(
+              value: value,
+              child: Text(value),
+            );
+          }).toList(),
+        ), 
+      );
+    }
+    else {
+      return Container(
+        margin: EdgeInsets.only(right: 40.0),
+        padding: EdgeInsets.symmetric(horizontal: 10.0),
+        decoration: BoxDecoration(
+          border: Border.all(color: Color(0xffBEBEBE)),
+          borderRadius: BorderRadius.circular(40),
+        ),
+        child: DropdownButton<String>(
+          value: _department,
+          isExpanded: true,
+          underline: Container(),
+          onChanged: (String value) async {
+            setState(() {
+              _department = value;
+            });
+          },
+          items: <String>['指定しない'].map<DropdownMenuItem<String>>((String value) {
+            return DropdownMenuItem<String>(
+              value: value,
+              child: Text(value),
+            );
+          }).toList(),
+        ), 
+      );
+    }
+  }
+    
+  Widget _lectureDropdown() {
+    if (_universitySelected && _departmentSelected) {
+      return Container(
+        margin: EdgeInsets.only(right: 40.0),
+        padding: EdgeInsets.symmetric(horizontal: 10.0),
+        decoration: BoxDecoration(
+          border: Border.all(color: Color(0xffBEBEBE)),
+          borderRadius: BorderRadius.circular(40),
+        ),
+        child: DropdownButton<String>(
+          value: _lecture,
+          isExpanded: true,
+          underline: Container(),
+          onChanged: (String value) {
+            setState(() {
+              _lecture = value;
+            });
+          },
+          items: lectureList.map<DropdownMenuItem<String>>((String value) {
+            return DropdownMenuItem<String>(
+              value: value,
+              child: Text(value),
+            );
+          }).toList(),
+        ), 
+      );
+    }
+    else {
+      return Container(
+        margin: EdgeInsets.only(right: 40.0),
+        padding: EdgeInsets.symmetric(horizontal: 10.0),
+        decoration: BoxDecoration(
+          border: Border.all(color: Color(0xffBEBEBE)),
+          borderRadius: BorderRadius.circular(40),
+        ),
+        child: DropdownButton<String>(
+          value: _lecture,
+          isExpanded: true,
+          underline: Container(),
+          icon: Icon(Icons.keyboard_arrow_down),
+          onChanged: (String value) async {
+            setState(() {
+              _lecture = value;
+            });
+          },
+          items: <String>['指定しない'].map<DropdownMenuItem<String>>((String value) {
+            return DropdownMenuItem<String>(
+              value: value,
+              child: Text(value),
+            );
+          }).toList(),
+        ), 
+      );
+    }
+  }
+
+  Widget _dropDownTitle (String value) {
```

Comment:
`dropdown` でググってみると1単語で表すことが一般的なようです。

```suggestion
  Widget _dropdownTitle (String value) {
```